integration-tests/src/test/add-ask-order.test.ts

```diff
@@ -0,0 +1,59 @@
+import { Guid } from 'js-guid';
+import { KeyringPair } from '@polkadot/keyring/types';
```

Comment:
From what I can tell CI is still using the old files to execute the integration tests. 

I don't see changes to `.github/workflows/ci.yml`. You will have to adjust which test files are used in the CI jobs before we can merge this PR.

Reply:
This is still a WIP. Once I have tests moved over I'll change CI.